src/NServiceBus.RavenDB.Tests/When_providing_a_custom_document_store.cs

```diff
@@ -14,7 +14,7 @@ public void Should_not_resolve_until_start()
             var endpointConfiguration = new EndpointConfiguration("custom-docstore-endpoint");
 
             endpointConfiguration.AssemblyScanner().ExcludeAssemblies("NServiceBus.RavenDB.Tests");
-            endpointConfiguration.UseTransport<LearningTransport>();
+            endpointConfiguration.UseTransport(new LearningTransport { TransportTransactionMode = TransportTransactionMode.ReceiveOnly });
```

Comment:
Why do we need to set the TX Mode?

Reply:
Otherwise you get the Outbox complaining about not being in ReceiveOnly exception. Adding a comment.

Reply:
Ah so AtomicSendsWithReceive isn't supported with the outbox?

Reply:
Yeah - seems stupid but when you remember how Outbox works, the incoming mesage has to be ACKed along with the database, and before outgoing messages are sent out. The sends can't be atomic with the receives.

Reply:
💡

Reply:
Wait a second, now that doesn't make sense to me! Let's talk about that.

Reply:
After https://github.com/Particular/NServiceBus/pull/5988is merged, this test will run just fine in the default SendsAtomic mode.